tools/c7n_azure/c7n_azure/resources/redis.py

```diff
@@ -42,3 +45,26 @@ class resource_type(ArmResourceManager.resource_type):
             'properties.sku.[name, family, capacity]'
         )
         resource_type = 'Microsoft.Cache/Redis'
+
+
+@Redis.filter_registry.register('redis-firewall-filter')
```

Comment:
re naming we try to avoid redundant names, ie not including the resource name in filter name.